src/store/reducers/swapReducer.ts

```diff
@@ -1,16 +1,41 @@
-import { SELECT_SWAP_DATE } from '../actions';
+import {
+    SELECT_SWAP_DATE,
+    SELECT_SWAP_OPTION,
+    SELECT_SWAP_TARGET,
+    SEND_SWAP_REQUEST,
+    RESET_SWAP_CONFIG,
+} from '../actions';
 
 const initialState = {
+    swapOption: 0,
```

Comment:
seeing that the reducer is called `swapReducer`, `swapReducer.date || swapReducer.target || swapReducer.option` is already descriptive enough

Reply:
changes made